src/runtime/components/forms/Form.ts

```diff
@@ -21,21 +21,21 @@ export default defineComponent({
     validate: {
       type: Function as PropType<(state: any) => Promise<FormError[]>> | PropType<(state: any) => FormError[]>,
       default: () => []
+    },
+    disableAutoValidation: {
```

Comment:
Instead of a Boolean here, we could do like `FormKit` and add a `validation-visibility` that defaults to `blur` so we can add more modes later on: https://formkit.com/essentials/validation.

What do you think?

Reply:
Good idea, shouldn't it be an array `:validate-on="['blur', 'change', 'input']"` to give more control?

Reply:
Indeed this would be even better!

Reply:
Could `submit` be done too?